### DISCUSSION TITLE
how to change file dumpwave by cocotb not by tool

### DISCUSSION THREAD

**Discussion by AnhHonag** [cocotb/cocotb#4133] [Q&A]

**Comment by imphil** [comment]
@AnhHonag Unfortunately, I'm unable to understand your question in that brevity. Please follow the guidance at https://github.com/cocotb/cocotb/issues/4108 for us to be answer your questions.

**Comment by ktbarrett** [comment]
Dupe. Closing in favor of #4134. Please stop making new Github issues; you can edit old ones.
